### PR TITLE
Set up Aqueduct for the rules builder server

### LINKED ISSUE #71: Set up Aqueduct for the rules builder server

### PR DESCRIPTION
**Pull request by thu5**
Generated the basic scaffolding for the server. Closes #71 .

### CODE REVIEW

File: server/lib/channel.dart [hunk 1]
```diff
@@ -0,0 +1,39 @@
+import 'server.dart';
```
Comment (ruben-aghayan) [inline server/lib/channel.dart:1]: is this a template file or from an example?

**Review by ruben-aghayan — COMMENTED** [review]
nit: newlines end of file

Jin: ^ dartfmt can help you check for that

File: server/lib/channel.dart (hunk 1 above)
Comment (thu5) [reply server/lib/channel.dart:1]: It's a template file.

**Review by ruben-aghayan — APPROVED** [review]

**Comment by cathyang-work** [comment]
Not familiar with Aqueduct, I will just acknowledge this pull request.

**Pull request merged**